docs/logs.md

```diff
@@ -2,4 +2,170 @@
 sidebar_position: 9
 ---
 
-# Logs
\ No newline at end of file
+# Logs Overview 📊
+
+## Query Search Interface Overview 🔍
+
+- Overview of the query search interface supporting SQL, SPL, and LogQL queries.
+
+
+
+https://github.com/Davleen14/siglens-docs/assets/114626340/134cf62c-2c4b-41bd-ab8e-1b64d3dfb6b1
+
+
+
+
+### Steps for Using the Query Search
+
+1. **Select or Enter Queries:**
+   - You can either select a query from the info icon or enter your own.
+   ![Query Search](/static/img/icon.png)
+
+2. **Choose Query Type:**
+   - Select which type of query you want to search: SQL, SPL, LogQL.
+   ![Query Type](/static/img/query-type.png)
+
+3. **Select Index:**
+   - Choose an index from the index dropdown.
+   ![Index](/static/img/index.png)
+
+4. **Set Time Range:**
+   - Select the time range for the query.
+   ![Time Range](/static/img/time.png)
+
+5. **Use Query Builder:**
+
+
+
+https://github.com/Davleen14/siglens-docs/assets/114626340/2255a445-462b-4c22-90cb-67078edeb9dd
+
+
+
+   ![Query Builder Search](/static/img/query-builder.png)
```

Comment:
this screenshot does not show anything specific so may be remove it. This and next screen shot is a bit bigger that the screenshots before.